scripts/SMD_chip_package_rlc-etc/size_definitions/size_default_chip.yaml

```diff
@@ -60,19 +62,20 @@ SMD_1210:
     terminator_spacing_max: 2.32
     size_info: '(Body size source: http://www.tortai-tech.com/upload/download/2011102023233369053.pdf)'
 
-SMD_1806:
-    code_imperial: "1806"
-    code_metric: "4516"
-    body_length:
-      nominal: 4.5
-      tolerance: 0.2
-    body_width:
-      nominal: 1.6
-      tolerance: 0.2
-    terminal_length:
-      nominal: 0.7
-      tolerance: 0.3
-    size_info: '(Body size source: https://www.modelithics.com/models/Vendor/MuRata/BLM41P.pdf)'
+#Could not find any diode, fuse, or LED in this body size
```

Comment:
The murata datasheet does have a ferrite bead in that size.
LEDs also exist: https://www.mouser.de/datasheet/2/216/APDA1806ZGCK-1374637.pdf
For Diodes I agree, at least a quick search did not return anything.

Reply:
This comment was already there, and it doesn't mention inductors (which should cover ferrite beads). As commented on line 126 of SMD_packages.yaml, for inductors there is a specific L_1806 definition in size_inductor.yaml using the IPC-SM782 body dimensions (annoying only given as 4516 and not showing the 1806 equivalency). Good catch on the LED; I will add it in a separate commit soon. You didn't mention anything about fuses but I quickly checked DigiKey and didn't see any with 1.8x0.6mm body size.

Reply:
The IPC-7351 fillet goals are very specific. The ones for rectangular-end components must be the one to use here, and this is the only reference to LEDs I could find (see 'LEDM' in section 8.4:
![image](https://user-images.githubusercontent.com/1936989/93030333-12c69780-f5d7-11ea-82c0-a26373cd6411.png)

I don't see anything that makes me think this LED footprint wouldn't work based on the image in the datasheet, as it has leads wrapping around just the ends like a chip resistor, so I added it. But like I said, I'm not 100% sure. The resulting pads are a bit smaller than shown in that datasheet but fairly close (0.85x0.75mm) so it seems OK.

Reply:
TBH i have no clue which footprint to use for a ferrite bead. I just had to look which fp-filters are set for the symbol.

Regarding the LED I made a mistake. It is a side-emitting LED. Its only `1806 Metric` when viewed from the side.
But it seems you fell for it as well
```yaml
LED_1806:
    code_imperial: "1806"
    code_metric: "4516"
    body_length:
      nominal: 1.8
      tolerance: 0.15
    body_width:
      nominal: 0.6
      tolerance: 0.1
    terminal_length:
      nominal: 0.3
      tolerance: 0.15
    ipc_reference: "ipc_spec_larger_or_equal_0603"
    size_info: '(Body size from: https://www.mouser.de/datasheet/2/216/APDA1806ZGCK-1374637.pdf)'
```
That does not make sense. `code_imperial` is 1806, but the metric size is 1806.

Reply:
Ahh! Yes. Good catch. I'll undo that.